README.md

```diff
@@ -1,28 +1,279 @@
 
+The xiSEARCH algorithm is described in detail in [Mendez, Fischer *et al.* Mol. Sys. Bio. 2019](https://www.embopress.org/doi/full/10.15252/msb.20198994). The xiSEARCH scoring function is made up of several terms accounting for the goodness of fit of the spectra to the peptide pair selected from the database, including fragment mass error, percentage intensity explained, number of fragments, number of crosslinked fragments.
 
-the Gui provides several tabs to configure your search. 
-In the first tab (Run) the result file (result) and optionally a for writing out full annotations of the spectra (peak annotations) can be defined.
+Scoring happens in three stages: 
+
+1. alpha candidates are selected and scored
+2. top n alpha candidates are taken and all matching beta-candidates (according to the precursormass) will be selected and prescored as pairs
+3. the top X of these are then fully matched and scored
+
+The scoring function is applied to explain each spectrum with a target-target pair, a target-decoy pair, a decoy-decoy pair, a linear target sequence or a linear decoy sequence. Error control by false discovery rate estimation is then performed in a separate step with xiFDR.
+
+
+
+### The interface
+the interface provides several tabs to configure your search. In the first tab (Run) the result file (result) and optionally a for writing out full annotations of the spectra (peak annotations) can be defined.
 
 In the second tab (Peak Lists) the peak lists to be searched can be defined. Supported formats are mgf-files as generated from msconvert or apl-files as produced by MaxQuant. For mgf-files of other sources the config might need to be adapted to define how to read out a run-name and a scan-number from the TITLE= tags in the files.
 
-The third tab defines the FASTA files to be searched. Several files can be defined. One also define one of the fasta files as custom decoy database. If no decoy database is given one is automatically generated based on the target database.
+The third tab defines the FASTA files to be searched. Multiple files can be defined. 
 
 The fourth tab provides the search configuration. Here all parameters are defined as a in form of a text file. A default config is provided as an example that defines a search with BS3 as cross-linker and Carboxyamidomethylation on Cystein as fixed modification and oxidation of methionine as variable modification. Each option contains a short description of what it does and how to change it.
 
 When everything is configured as desired then start-button on the first tab can be pressed to start the search.
```

Comment:
That whole section is of an old version. and probably should go

Reply:
then it goes :)